pages/feedingtime.js

```diff
@@ -0,0 +1,97 @@
+import { useSession, signIn } from "next-auth/react";
+import { unstable_getServerSession } from "next-auth/next";
+import { authOptions } from "./api/auth/[...nextauth]";
+import styled from "styled-components";
+import { CardContainer } from "../components/CardContainer";
+import Card from "../components/Card";
+import { getAllDays } from "../services/dayService";
+import { SignInButton } from "../components/SignInButton";
+
+import { CanvasContainer } from "../components/CanvasContainer";
+import LineChart from "../components/charts/LineChart";
+
+export async function getServerSideProps(context) {
+  const session = await unstable_getServerSession(
+    context.req,
+    context.res,
+    authOptions
+  );
+  if (session) {
+    const days = await getAllDays(session.user.email);
+    return {
+      props: { days: JSON.parse(JSON.stringify(days)) },
+    };
+  } else return { props: {} };
+}
+
+export default function FeedingTimePage({ days }) {
+  //----- Session -----
+  const { data: session } = useSession();
+
+  if (session) {
+    const filteredDays = days.filter((day) =>
+      Boolean(day.feastTimes.length > 0)
```

Comment:
I don't think you need the Boolean here since the expression already returns a boolean value.

Reply:
yeah, I tried it and it works perfectly, thank you!!